proxy/config.go

```diff
@@ -312,6 +313,12 @@ func LoadConfigFromReader(r io.Reader) (Config, error) {
 				if macroName == "PID" && fieldName == "cmdStop" {
 					continue // this is ok, has to be replaced by process later
 				}
+				if macroName == "MODEL_ID" {
```

Comment:
I think it would be better to move MODEL_ID replacement outside of the loop and make it always run on Cmd and CmdStop. I suggest putting the `${MODEL_ID}` replacement right after the `${PORT}` replacement and leaving the search for unknown macros the same as before.